packages/itwinui-css/vite.config.ts

```diff
@@ -18,7 +25,7 @@ export default defineConfig({
       },
     },
   },
-  plugins: [lightningCssPlugin(), generateIndex(), addMetaTags(), minifyHtml()],
+  plugins: [generateIndex(), addMetaTags(), minifyHtml()],
```

Comment:
this is the _only_ change in this file.
```diff
-  plugins: [lightningCssPlugin(), generateIndex(), addMetaTags(), minifyHtml()],
+  plugins: [generateIndex(), addMetaTags(), minifyHtml()],
```

everything else happened automatically through prettier / copyrightLinter.

i think this is because it is now using the shared prettier config whereas previously it was using editorconfig (which was removed in #868). they have different line lengths (#280)

the copyright change makes sense but we might want to introduce a new prettier config for the css workspace to avoid formatting changes like this.

what's also interesting is that we run linter in CI and it was passing, meaning something is not working there. need to investigate this in a separate PR